apps/examples/testcase/ta_tc/systemio/itc/itc_gpio.c

```diff
@@ -1130,6 +1130,42 @@ static void itc_systemio_gpio_get_drive_mode_n_after_close(void)
 	TC_SUCCESS_RESULT();
 }
 
+static void itc_systemio_gpio_register_signal_p(void)
```

Comment:
we need to remove itc as well, as we want to remove both utc and itc codes together. Need to crate new generic directory

Reply:
done. please check now